client/a8c-for-agencies/sections/sites/features/hosting/style.scss

```diff
@@ -1,3 +1,64 @@
-.site-preview-pane__hosting-content {
-	margin: 32px 0;
+.hosting__card {
+	border-radius: 4px;
+	padding: 0;
+}
+
+.hosting__header {
+	border-bottom: 1px solid var(--color-border-subtle);
+}
+
+.hosting__title {
+	padding: 16px 24px;
+	font-size: rem(16px);
+	font-weight: 600;
+	line-height: 24px;
+	text-align: left;
+}
+
+.hosting__content {
+	display: grid;
+	grid-template-columns: auto 1fr;
+	text-align: left;
+	padding: 8px 0;
+}
+
+.hosting__content-row {
+	display: contents;
+}
+
+.hosting__content-value,
+.hosting__content-label {
+	padding: 12px 24px;
+}
+
+
+.hosting__content-label {
+	color: var(--studio-gray-60, #50575e);
```

Comment:
Let's use `--color-accent-60`

Reference for the tip: https://github.com/Automattic/wp-calypso/pull/89081/files#r1549034432

Reply:
Changed in 2a3941e. Thanks for the reference!